lusidtools/cocoon/cocoon_printer.py

```diff
@@ -0,0 +1,159 @@
+import pandas as pd
+import numpy as np
+
+
+def check_errors_and_success(target_dict: dict, target_name: str, required_keys: list):
+    """
+    This function checks that a named dictionary contains a list of required keys
+    :param dict target_dict: Dictionary to check for keys in
+    :param str target_name: Variable name of dictionary
+    :param list required_keys: List of required keys
+    :return:
+    """
+    for key in required_keys:
+        if key not in target_dict.keys():
+            raise ValueError(f"'{key}' missing from {target_name}. {target_name} must include 'errors', and 'success' "
+                             f"as keys")
```

Comment:
Change this to be a generic message including the provided keys.

Reply:
````raise ValueError(f"'{key}' missing from {target_name}. {target_name} must include {required_keys}"
                             f"as keys")
```